tracer/test/snapshots/Security.AspNetWebApi.Integrated.enableSecurity=False.__test=blocking.verified.txt

```diff
@@ -0,0 +1,147 @@
+[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: aspnet.request,
+    Resource: GET /health,
+    Service: sample,
+    Type: web,
+    Tags: {
+      env: integration_tests,
+      http.client_ip: 127.0.0.1,
+      http.method: GET,
+      http.request.headers.host: localhost:00000,
+      http.status_code: 404,
```

Comment:
The difference between AspNetWebApi.Classic.enableSecurity=False and AspNetWebApi.Integrated.enableSecurity=False is this one reports a tag `http.status_code: 404`. Is this correct?

Reply:
Good spot, thanks! It's seems I was using the wrong url in the web api tests, it should be prefixed with `/api`. 

Fixed in ddae81258